Client/UserResearch/NewTabUserResearch.swift

```diff
@@ -54,25 +63,31 @@ class NewTabUserResearch {
                 return
             }
             self.fetchedExperimentVariables = true
-            self.updateTelemetry()
-            self.newTabState = LPVariables.newTabButtonABTest?.boolValue() ?? false
+            //Only update add new tab (+ button) when it doesn't match leanplum value 
+            let lpValue = LPVariables.newTabButtonABTest?.boolValue() ?? false
+            if self.newTabState != lpValue {
+                self.newTabState = lpValue
+                self.updateTelemetry()
+            }
         }
         // Condition: Leanplum server too slow; Set default New tab state
         DispatchQueue.main.asyncAfter(deadline: .now() + 2) {
-            guard self.fetchedExperimentVariables == false else {
+            guard self.fetchedExperimentVariables == false && self.newTabState == nil else {
```

Comment:
Added additional step to only go here if LP was too slow and we did not have anything assigned to `newTabState` variable, otherwise we'll keep on going to this condition everytime LP is slow which isn't really required.